example/ionic-angular-v2/android/app/src/main/assets/capacitor.config.json

```diff
@@ -2,6 +2,9 @@
   "appId": "io.ionic.starter",
   "appName": "ionic-angular",
   "bundledWebRuntime": false,
+  "ios": {
+    "minVersion": "12.0"
```

Comment:
Is this an SDK imposed limit? If so, we should note this on the docs

Reply:
Nope, this is a Capacitor 3 requirement. It still works with capacitor 2 but they'll either need to bump their version up to 12 like in this config json, or they can write a script in their pod file to force it back down.